.github/workflows/cli_core_e2e_sample-plugin.yaml

```diff
@@ -0,0 +1,56 @@
+name: Tanzu CLI Sample-Plugin E2E Tests
+
+on:
+  pull_request:
+    branches: [main, release-*]
+  push:
+    branches: [ main, test*, release-*]
+
+jobs:
+  build:
+    name: Tanzu CLI Sample-Plugin E2E Tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
```

Comment:
I am not sure we need to run these tests on all OS Arch. If there is no good reason, we can just run these on `ubuntu` for now.

Reply:
updated to run only on ubuntu.